src/plus/webviews/graph/protocol.ts

```diff
@@ -174,6 +175,11 @@ export const UpdateRefsVisibilityCommandType = new IpcCommandType<UpdateRefsVisi
 	'graph/refs/update/visibility',
 );
 
+export interface SwitchToRefParams {
+	ref: GraphRef
+}
+export const SwitchToRefCommandType = new IpcCommandType<SwitchToRefParams>('graph/ref/checkout');
```

Comment:
Reframe this to be `DoubleClickedRefCommandType`, so that why on the GL side we can decide the behavior of the double click